app/src/User/UsernameInputFormType.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace User;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Form used to accept input of an email address only
```

Comment:
Comment should say "of a username only" - I'll update on merge, but putting here as a reminder to myself!